### PR TITLE
{bio}[goolf/1.7.20] deepTools v2.3.5

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
+# this application also requires numpy, scipy and matplotlib which
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb]: @pescobar `matplotlib` is *not* part of the upstream Python easyconfigs...

File: easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
+# this application also requires numpy, scipy and matplotlib which 
+# are provided by the Python module 
+exts_list = [
+    ('six', '1.10.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb]: this one *is* included in Python already

File: easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('dateutil', '2.5.3', {
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb]: already included

File: easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        'source_tmpl': 'python-%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-dateutil/'],
+    }),
+    ('pytz', '2016.6.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/d/deepTools/deepTools-2.3.5-goolf-1.7.20-Python-2.7.11.eb]: already included?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2051.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/62810a382e0f5a6c2e784f710952dc3e for a full test report.

**Comment by boegel** [comment]
@pescobar missing dep on `cURL`?

```
Either libcurl isn't installed, it didn't come with curl-config, or curl-config isn't in your $PATH. This must be corrected before installing pyBigWig!
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 11 out of 12 (4 easyconfigs in this PR)
node2568.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dd02c02d5dc0fff6cb5fd1921e9335b9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 5 (4 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b45a53c0a1626ee4423482cb1b95a1e2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2134.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6094d35b24e868b9b6e97174adc4907b for a full test report.

**Comment by boegel** [comment]
@pescobar more missing deps:

```
ImportError: No module named sphinx
```

You'll need to add `sphinx` to `exts_list` as well?

**Comment by boegel** [comment]
@pescobar ping on adding `sphinx` too?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**